# base path for floor images
# string
base_path: "/home/robot/floors/"
